added get_channel_id method

sumzer opened the pull request:
We are using this repo for our project and we need method getChannelId wich is provided by UrbanAirship SDk library.
http://docs.urbanairship.com/platform/android.html#retrieving-your-channel-id
can you check it and approve it.

Thanks in advance
Best regards 
Sumi

sumzer committed:
added get_channel_id method

rlepinski commented on android/src/main/java/com/globo/reactnativeua/ReactNativeUA.java:
```diff
@@ -55,6 +55,12 @@ public void removeTag(String tag) {
 
     @ReactMethod
+    public void getChannelId(Callback callback) {
+	     String channelId = UAirship.shared().getPushManager().getChannelId();
```
spacing is off here

rlepinski commented:
Could you update the README with the added method?

Other than the code spacing issue, everything looks good to me.

sumzer committed:
added description into Readme file
fix for indentation
changed position of get_channel_id method in index.js file, because of linking code desc in readme file

sumzer commented:
Hi,
I've pushed changes, please check it.
Best regards
Sumi

rlepinski commented:
This looks great to me. I do not have the ability to merge commits. Paging @evandroeisinger

evandroeisinger commented:
@rlepinski you didn't have? We already gave you write permissions; Could you check, please?

rlepinski commented:
@evandroeisinger I had an invite in my inbox from a few weeks ago. Thanks :)

evandroeisinger commented:
@rlepinski just to let you know that I already added you to the npm project. Now you are able to deploy new versions.
